_posts/2017-04-24-a-matter-of-perception-and-context.md

```diff
@@ -0,0 +1,79 @@
+So, I started going down a different train of thought. Why not embrace the opportunities of the digital era? Share a bit about me, my perceptions and my personal experiences. Consider that there are seven billion different perspectives on the human experience. 
+It's an extraordinary thought!
+
+## Domain Driven Design
+
+Back to my topic of context, I want to share my appreciation for Domain Driven Design, with my focus mainly on the value of establishing and using an "ubiquitous language". This supports a significant part of DDD's strategic design and the "Bounded Context" pattern.
+ 
+It's addressing an age old problem of communicating concepts and ensuring a shared understanding of it. I've often wondered secretly by myself after walking out of a meeting room if everyone leaving the door have the exact same understanding of what exactly has been agreed upon. What are the odds?
+
+My favourite example is the abbreviation CBT. Someone from an educational domain may think "Computer Based Training"; another from Human Psychology domain may think "Cognitive Behavioural Therapy"; and on the odd chance there was a person with a certain fetish in the audience, well, that would be quite another ball game all together!
+
+Many project teams in the past had to learn from their mistakes the hard way. As the saying goes, rather learn from other's mistakes than your own. It's less painful that way.
+
+### Context
+
+**Definition**
+*The setting in which a word or statement appears that determines its meaning.
+Statements about a model can only be understood in a context.*
+
+### Bounded Context Pattern
+
+**Definition**
+*A description of a boundary (typically a subsystem, or the work of a particular team) within which a particular model is defined and applicable.*
+
+As with the CBT example, the term can only be understood correctly, within context, once the boundary has been defined explicitly.
+
+All players in the project team will then be bound to this context to:  
+   1. specify business requirements  
+   2. design the domain model  
+   3. name your code  
+
+### Ubiquitous Language
+
+**Definition**
+*A language structured around the domain model and used by all team members within a bounded context to connect all the activities of the team with the software.*
+
+Once the bounded context has been defined, all collaboration and communication should be committed to using this language. 
+This is where the real benefit lies, as people will develop a shared view of the concepts.  
+And voilà! No more misunderstandings.
+
+## Final Words
+
+Domain Driven Design serves as guide with principles, patterns and philosophies. My purpose here was to raise awareness. 
+At this point I'll hand you over to the experts. For further reading, my personal favourites are Eric Evans and Scott Millett.
```

Comment:
Worth giving Vaughn Vernon a nod as well since he's the author of one of the two best know books on DDD.